src/client/components/MyProfile/ProfilePage/ProfileComponent.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { handlerUserContext } from '../../../App';
+import './ProfileComponent.styles.css';
+import { ProfileCardComponent } from '../../ProfileCardComponent/ProfileCardComponent';
+import { YourUploadedCVs } from '../YourUploadedCVs/YourUploadedCVs';
+import TitleCvCard from '../TitleCvCard/TitleCvCard';
+
+export default function ProfileComponent() {
+  const { setUserName } = useContext(handlerUserContext);
+  const [cvsList, setCvsList] = useState([]);
+  const [userData, setUserData] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
```

Comment:
Shall I change this method of get errors? to get rid of the following failure?

Reply:
You can remove the line, since it's not being used.

Reply:
@pdsorensen So does it mean in line 28 ```setErrorMessage``` never used? and the catch error here is useless?

Reply:
`errorMessage` is not being used. You can do: `[_, setErrorMessage] ...`, or display the error message.